src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -106,9 +139,19 @@ export function AdvancedSwapDetails({ trade, allowedSlippage, syncing = false }:
         </RowBetween>
         {!trade?.gasUseEstimateUSD || !chainId || !SUPPORTED_GAS_ESTIMATE_CHAIN_IDS.includes(chainId) ? null : (
           <RowBetween>
-            <ThemedText.SubHeader color={theme.text3}>
-              <Trans>Network Fee</Trans>
-            </ThemedText.SubHeader>
+            <MouseoverTooltip
+              text={
+                <Trans>
+                  The fee paid to miners who process your transaction. This must be paid in{' '}
+                  {CHAIN_INFO[chainId ?? 1].nativeCurrency.symbol}.
```

Comment:
Instead of using `CHAIN_INFO`, use `useNativeCurrency()`.symbol.

Reply:
Done.